Library/Homebrew/requirements/java_requirement.rb

```diff
@@ -129,7 +129,7 @@ def oracle_java_os
   end
 
   def satisfies_version(java)
-    java_version_s = system_command(java, args: ["-version"], print_stderr: false).stderr[/\d+\.\d/]
+    java_version_s = system_command(java, args: ["-version"], print_stderr: false).stderr[/\d+(\.\d)?/]
```

Comment:
The second number could be > 9, so let's also use `\d+` for it.

Reply:
OK, done.  I've never seen a java minor version > 9 (only ..., 1.6, 1.7, 1.8, 9.0, 10.0, 11.0, 12) but that's easy to account for.  My only question in this hypothetical scenario would be whether the existing code handles any "+" comparison correctly, e.g. would it treat "1.11" as > "1.8"?

Reply:
As long as the existing code uses the `Version` class, comparisons are handled appropriately.